### PR TITLE
docs: Web in-progress builds

### LINKED ISSUE #1176: web: design in-progress builds related changes
**Description**
Desing changes related to the in-progress builds.

**Acceptance Criteria**
- [ ] describe the required changes to the application
- [ ] class diagrams

**Additional context**
https://github.com/platform-platform/monorepo/blob/master/metrics/ci_integrations/docs/06_builds_synchronization.md

### PR DESCRIPTION
**Pull request by solid-vladlaba**
Fixes #1176 

## Checklist

- [x] Documented

### CODE REVIEW

File: metrics/web/docs/features/in_progress_builds/diagrams/domain_layer_class_diagram.puml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            - _repository: MetricsRepository
+            + call(ProjectIdParam): Stream<DashboardProjectMetrics>
+            ..
+            - _getPerformanceMetrics(List<Build> builds): PerformanceMetric
+            - _getStability(List<Build> builds): Percent
+        }
+
+        ReceiveProjectMetricsUpdates -down-> ProjectIdParam: uses
+        ReceiveProjectMetricsUpdates --> DashboardProjectMetrics: creates
+    }
```
Comment (solid-vovabeloded) [inline metrics/web/docs/features/in_progress_builds/diagrams/domain_layer_class_diagram.puml:87]:
```suggestion
        ReceiveProjectMetricsUpdates --> DashboardProjectMetrics: creates
        ReceiveProjectMetricsUpdates --> MetricsRepository : uses
    }
```
It seems like this relation is missing.

**Review by solid-vovabeloded — CHANGES_REQUESTED** [review]
Excellent job, @solid-vladlaba! I've added a couple of suggestions here - please review them. Overall, the document looks great and easy-to-read, and the diagrams clearly describe the required changes.

**Review by solid-vovabeloded — APPROVED** [review]
LGTM, thanks @solid-vladlaba! The document looks awesome!

**Comment by solid-vladlaba** [comment]
@Mergifyio refresh

**Pull request merged**